pyproject.toml

```diff
@@ -100,9 +103,7 @@ testpaths = [ "tests" ]
 branch = true
 relative_files = true
 omit = [ "nox/_typing.py" ]
-
-[tool.coverage.report]
-exclude_lines = [ "pragma: no cover", "if TYPE_CHECKING:", "@overload" ]
+source_pkgs = [ "nox" ]
 
```

Comment:
To merge between different base paths, I believe you want something like the following (this is what we use in pyca/cryptography -- it needs adaptation for nox of course0):

```
[tool.coverage.paths]
source = [
   "src/cryptography",
   "*.nox/*/lib*/python*/site-packages/cryptography",
   "*.nox\\*\\Lib\\site-packages\\cryptography",
   "*.nox/pypy/site-packages/cryptography",
]
tests =[
   "tests/",
   "*tests\\",
]
```

Reply:
FYI, I don't think `tool.coverage.paths.tests` is a valid entry.

Reply:
I believe the keys can have any names you want -- the structure is that teh first item in the array is the "normalized value" and the other values are prefixes that normalize to it. https://coverage.readthedocs.io/en/7.4.2/config.html#paths

Reply:
Oh, missed that, nice!